### ISSUE TITLE
Current model

### ISSUE THREAD

**Issue by SnorreFossland** [Kavca/kavca-dementia-models#9]

**Comment by SnorreFossland** [comment]
# 2023-04-04 

![Image](https://user-images.githubusercontent.com/31763773/229845225-b0162814-f87d-475b-9b0e-b35b48902f14.png)

**Comment by SnorreFossland** [comment]
# Dementia Diagnosis process

![Image](https://user-images.githubusercontent.com/31763773/229846078-f60f3c4c-fa32-484e-9b9b-60f72a48d08e.png)

**Comment by JorgenFagerquist** [comment]
Would a process diagram (IDEF/BPMN) be appropriate in this instance?

_________________________________________________________________________

Respectfully,

Jörgen Fägerquist | (202) 380-6127 |

 ***@***.***> ***@***.*** |

511 Lisa Ave | Odenton | MD | 21113-2021

**Comment by SnorreFossland** [comment]
Yes, I think so.  This Current model is more of a Scaffolding model (some call it a Prime model), attempting to flush out the main terms, object-types needed for a Dementia  treatment model. Next step is to identify and define the object-types needed  in what we call a Concept model. Then we can generate a metamodel for solution modelling, where IDEF/BPMN is alternative views.
It can however be useful in the Scaffolding fase, to make example IDEF/BPMN modelviews in order to clarify certain tasks/processes/activities.
